pomegranate/hmm.pyx

```diff
@@ -29,6 +29,7 @@ from .callbacks import History
 
 from .utils cimport _log
 from .utils cimport pair_lse
+from .utils import check_random_state
 
```

Comment:
Can't seem to comment on line 883 in github.
I would like to comment that at least a comment is in order for that line.
There are serious repercussions to setting the seed to 0 as a side-effect of bake.
 https://github.com/jmschrei/pomegranate/pull/439/files#diff-16503312cc32dfb87898df1d62fd1d8bR881